plugins/javaembed/javaembed.cpp

```diff
@@ -3790,8 +3853,14 @@ class JavaEmbedImportContext : public CInterfaceOf<IEmbedFunctionContext>
     virtual void importFunction(size32_t lenChars, const char *utf) override
     {
         if (!javaClass)
-            loadFunction(lenChars, utf, classpath, 0, nullptr);
-        reinit();
+        {
+            size32_t bytes = rtlUtf8Size(lenChars, utf);
+            importName.set(utf, bytes);
+            if (strchr(importName, '.'))  // !isIntanceParameter)
```

Comment:
typo intance->instance

Reply:
I should probably just remove the parameter